src/ext/FirewallExtension/ca/firewall.cpp

```diff
@@ -39,7 +39,7 @@ static UINT SchedFirewallExceptions(
     LPWSTR pwzCustomActionData = NULL;
     LPWSTR pwzName = NULL;
     LPWSTR pwzRemoteAddresses = NULL;
-    int iPort = 0;
+    LPWSTR pwzPort = NULL;
```

Comment:
I have added the missing ReleaseStr and have signed the agreement on January 9. But I have not enough WiX programming knowledge/skills to program the parameter validation.